pkg/network/tracer/tracer.go

```diff
@@ -62,15 +63,17 @@ const tracerModuleName = "network_tracer"
 // If we want to have a way to track the # of active TCP connections in the future we could use the procfs like here: https://github.com/DataDog/datadog-agent/pull/3728
 // to determine whether a connection is truly closed or not
 var tracerTelemetry = struct {
-	skippedConns     telemetry.Counter
-	expiredTCPConns  telemetry.Counter
-	closedConns      *nettelemetry.StatCounterWrapper
-	connStatsMapSize telemetry.Gauge
+	skippedConns         telemetry.Counter
+	expiredTCPConns      telemetry.Counter
+	closedConns          *nettelemetry.StatCounterWrapper
+	connStatsMapSize     telemetry.Gauge
+	payloadSizePerClient telemetry.Gauge
 }{
 	telemetry.NewCounter(tracerModuleName, "skipped_conns", []string{}, "Counter measuring skipped TCP connections"),
 	telemetry.NewCounter(tracerModuleName, "expired_tcp_conns", []string{}, "Counter measuring expired TCP connections"),
 	nettelemetry.NewStatCounterWrapper(tracerModuleName, "closed_conns", []string{}, "Counter measuring closed TCP connections"),
 	telemetry.NewGauge(tracerModuleName, "conn_stats_map_size", []string{}, "Gauge measuring the size of the active connections map"),
+	telemetry.NewGauge(tracerModuleName, "payload_size_per_client", []string{}, "Gauge measuring the size of the delta conns per clientID"),
```

Comment:
I think `conn_payload_size` would be better. No need to reference the per client part. Description should just say "Gauge measuring the number of connections in the system-probe payload".

Reply:
thanks for the recs! Just updated